packages/turf-along/test.js

```diff
@@ -17,15 +20,31 @@ test('turf-along', t => {
     const pt7 = along(line, 100, options);
     const pt8 = along(line.geometry, 0, options);
     const fc = featureCollection([pt1, pt2, pt3, pt4, pt5, pt6, pt7, pt8]);
-
+    
     fc.features.forEach(f => {
         t.ok(f);
         t.equal(f.type, 'Feature');
         t.equal(f.geometry.type, 'Point');
     });
     t.equal(fc.features.length, 8);
-    t.equal(fc.features[7].geometry.coordinates[0], pt8.geometry.coordinates[0]);
-    t.equal(fc.features[7].geometry.coordinates[1], pt8.geometry.coordinates[1]);
+  
+    t.equal(isPointOnLine(fc.features[5].geometry.coordinates, line), true);
```

Comment:
Wouldn't this be the same, but more clean?
```js
fc.features.forEach(f => {
    t.true(isPointOnLine(f, line));
});
```

Reply:
This is the more accurate check I'd like to move towards, but for some reason some of the points are falling off the line (by a minutia), I'm not sure if this is a rounding thing or what...

Reply:
is the test line a rhumb or a great-circle line where it fails? Maybe we are now facing the reverse problem we want to solve with this fix.
I was wondering if moving everything to rhumb functions would actually fail in those other cases where conventional (I don't have a better word for this) functions were actually ok.

Reply:
Just double checked but the original approach also failed with the same test. I'm thinking it's something to do with floating point math that is causing us to fail on the test.